### PR TITLE
Upgrade to new version, Apply patch

### PR DESCRIPTION
**Pull request by agafandrey**

### CODE REVIEW

File: cookbook/how_to_apply_patch.rst [hunk 1]
```diff
@@ -116,3 +116,7 @@ Uncomment this line.
 
     $ sudo -uwww-data app/console lexik:maintenance:unlock --env prod
 
+    .. note::
+        If There are PHP opcache tools are in use, PHP-FPM (or Apache web server) should be restarted after uprgade
```
Comment (iklymenko) [inline cookbook/how_to_apply_patch.rst]: Replace: "opcache" => "bytecode cache (e.g. opcache)"

File: cookbook/how_to_upgrade_to_new_version.rst [hunk 2]
```diff
@@ -116,6 +116,10 @@ Uncomment this line.
 
     $ sudo -uwww-data app/console lexik:maintenance:unlock --env prod
 
+    .. note::
+        If There are PHP opcache tools are in use, PHP-FPM (or Apache web server) should be restarted after uprgade
```
Comment (iklymenko) [inline cookbook/how_to_upgrade_to_new_version.rst]: Replace: "opcache" => "bytecode cache (e.g. opcache)"

File: cookbook/how_to_upgrade_to_new_version.rst [hunk 3]
```diff
@@ -224,4 +228,8 @@ Uncomment this line.
...
```
Comment (iklymenko) [inline cookbook/how_to_upgrade_to_new_version.rst]: Replace: "opcache" => "bytecode cache (e.g. opcache)"

File: cookbook/how_to_apply_patch.rst [hunk 4]
```diff
@@ -116,3 +116,7 @@ Uncomment this line.
 
     $ sudo -uwww-data app/console lexik:maintenance:unlock --env prod
 
+    .. note::
+        If There are PHP bytecode cache (e.g. opcache) tools are in use, PHP-FPM (or Apache web server) should be
+        restarted after uprgade to flush cached bytecode from previous installation.
```
Comment (nnenasheva) [inline cookbook/how_to_apply_patch.rst]: Edit: If PHP bytecode cache tools (e.g. opcache) are used, PHP-FPM (or Apache web server) should be restarted after the uprgade to flush cached bytecode from the previous installation.

File: cookbook/how_to_upgrade_to_new_version.rst [hunk 5]
```diff
@@ -116,6 +116,10 @@ Uncomment this line.
 
     $ sudo -uwww-data app/console lexik:maintenance:unlock --env prod
 
+    .. note::
+        If There are PHP bytecode cache (e.g. opcache) tools are in use, PHP-FPM (or Apache web server) should be
+        restarted after uprgade to flush cached bytecode from previous installation.
```
Comment (nnenasheva) [inline cookbook/how_to_upgrade_to_new_version.rst]: Edit: Edit: If PHP bytecode cache tools (e.g. opcache) are used, PHP-FPM (or Apache web server) should be restarted after the uprgade to flush cached bytecode from the previous installation.

File: cookbook/how_to_upgrade_to_new_version.rst [hunk 6]
```diff
@@ -224,4 +228,8 @@ Uncomment this line.
...
```
Comment (nnenasheva) [inline cookbook/how_to_upgrade_to_new_version.rst]: Edit: If PHP bytecode cache tools (e.g. opcache) are used, PHP-FPM (or Apache web server) should be restarted after the uprgade to flush cached bytecode from the previous installation.

File: cookbook/how_to_apply_patch.rst [hunk 7]
```diff
@@ -116,3 +116,7 @@ Uncomment this line.
 
     $ sudo -uwww-data app/console lexik:maintenance:unlock --env prod
 
+    .. note::
```
Comment (DimaSoroka) [inline cookbook/how_to_apply_patch.rst]: Please check rendering of note, because of space it may be broken

Comment (nnenasheva) [inline cookbook/how_to_apply_patch.rst]: If you mean the space before "note", this is the rendering that we usually use.

**Pull request merged**